Teamwork (Graded)

Sebastian-Santana-Ort opened the pull request:
Merging my original teamwork branch back to the main @shycw1.

Sebastian-Santana-Ort committed:
HEre I am adding a .md teamwork file. I created then deleted a .Rmd file as it was not necessary.

Sebastian-Santana-Ort committed:
Deleted unnecesaary file.

johnnywang3739 commented:
Will merge this to the main (changes made by Sebastian about the teamwork contract), but there seems to be a conflict
- Chunyu Wang
19 Sep

Sebastian-Santana-Ort committed:
Merge branch 'main' into teamwork

Sebastian-Santana-Ort committed:
Updated .gitignore and readme.md to resolve merge conflict

Merge branch 'main' of https://github.com/stat545ubc-2023/collaborative-group24 into teamwork

# Conflicts:
#	.gitignore
#	TEAMWORK.md

johnnywang3739 reviewed (approved):
All good - Chunyu Wang

johnnywang3739 commented:
All looks good in teamwork contract. Change Merged
